nilearn/glm/first_level/first_level.py

```diff
@@ -124,26 +147,54 @@ def run_glm(Y, X, noise_model='ar1', bins=100, n_jobs=1, verbose=0):
     # Create the model
     ols_result = OLSModel(X).fit(Y)
 
-    if noise_model == 'ar1':
-        # compute and discretize the AR1 coefs
-        ar1 = (
-            (ols_result.residuals[1:]
-             * ols_result.residuals[:-1]).sum(axis=0)
-            / (ols_result.residuals ** 2).sum(axis=0)
-        )
+    if 'ar' in noise_model:
+
+        err_msg = ("AR order must be a positive integer specified as arN, "
+                   "where N is an integer. E.g. ar3. "
+                   f"You provied {noise_model}.")
```

Comment:
I think you're getting a lot of errors with Python 3.5 tests because of this fstring. 
Note: We'll drop Python 3.5 support in release 0.8, such that using fstrings will be fine then!

Reply:
Thanks! Old habits 😄 Should be fixed now.